Assets/Engine/Inventory/UI/ContainerUi.cs

```diff
@@ -16,6 +16,7 @@ public Inventory Inventory
 
         public AttachedContainer AttachedContainer
         {
+            get => attachedContainer;
```

Comment:
Out of curiousity, do you actually use this function?

Reply:
I used it to compare the attachedContainer in the containerUI and those retrieve doing a find but I don't anymore, I'm going to revert this change too